README.md

```diff
@@ -27,6 +27,52 @@ const MyComponent = () => (
 
 ```
 
+### Importing Sass Modules
+
+Example using Webpack:
+
+```js
+module.exports = {
+  ...,
+  resolve: {
+    alias: {
+      "@Feathers": path.resolve(__dirname, "node_modules/srf-feathers/build/foundation")
+    },
+    extensions: [".scss"],
+    module: {
+      rules: [
+        {
+          test: /\.(scss)$/i,
+          use: ["style-loader", "css-loader", "resolve-url-loader", "sass-loader"],
+        }
+      ]
+    }
+  }
+}
+```
+
+Import Feathers Modules in your stylesheet:
+
+```scss
+@import "@Feathers/breakpoints";
```

Comment:
...und dann `@Feathers/foundation/breakpoints`

Reply:
Angepasst in https://github.com/mmz-srf/feathers/commit/9da704e15fd9c8ea3462097049e2aefdf755756f